.gitignore

```diff
@@ -7,3 +7,5 @@ test_project/simple_api/testcases/models.py
 test_project/simple_api/testcases/objects.py
 test_project/simple_api/testcases/tests.py
 test_project/simple_api/testcases/permissions.py
+
+.idea/*
```

Comment:
tohle si dej k sobe do globalniho gitignore, sem to nepatri

Reply:
ja myslím že to sem kľudne patrí.. IDE tmp files welcome..  ale pohoda

Reply:
hehe, ja si myslim, ze ne :-D nemusis rikat svetu, ktere ide pouzivas, potom nechces mit zaznam v gitignore za kazdeho developera co pouziva nejake jine ide, a potom mas globalni gitignore u sebe, ktery slouzi presne k tomuto